HoloJS/HoloJsHost/ScriptingFramework/WebGL.js

```diff
@@ -307,6 +307,15 @@
         this.uniformMatrix4fv = function (location, transpose, value) {
             return nativeInterface.webgl.uniformMatrix4fv(this.glContext, location.external, transpose, value);
         }
+        this.stencilFunc = function (func, ref, mask) {
+            return nativeInterface.webgl.stencilFunc(this.glContext, func, ref, mask);
+        }
```

Comment:
Missing ';'

They are missing from almost everywhere in this file; we should probably fix them now.